projects/addon-commerce/components/input-card-group/input-card-group.component.ts

```diff
@@ -307,7 +313,8 @@ export class TuiInputCardGroup
         this.updateBin(bin);
 
         if (this.cardValidator(this.card) && !value()?.expire && this.inputExpire) {
-            this.focusExpire();
+            // Safari autofill focus jerk workaround
+            setTimeout(() => this.focusExpire());
```

Comment:
please add runOutsideAngular

Reply:
Doesn't matter, focus will trigger change detection it anyway